src/altrep.cc

```diff
@@ -40,15 +59,13 @@ bool vroom_altrep(SEXP x) {
 [[cpp11::register]] SEXP vroom_materialize(SEXP x, bool replace) {
 #ifdef HAS_ALTREP
   for (R_xlen_t col = 0; col < Rf_xlength(x); ++col) {
-
     SEXP elt = VECTOR_ELT(x, col);
-    // First materialize all of the non-character vectors
```

Comment:
I think this mention of "non-character vectors" was leftover from a version where there were two branches. Removal felt better than correction.